docs/3.0.0-beta.x/guides/registering-a-field-in-admin.md

```diff
@@ -65,12 +143,28 @@ const Wysiwyg = ({
   onChange,
   value,
 }) => {
+  const [isOpen, setIsOpen] = useState(false);
   let spacer = !isEmpty(inputDescription) ? <div style={{ height: '.4rem' }} /> : <div />;
 
   if (!noErrorsDescription && !isEmpty(errors)) {
     spacer = <div />;
   }
 
+  const handleChange = data => {
+    console.log(data);
```

Comment:
console.log

Reply:
I thought it was useful for the user to add this but I'll remove it